### ISSUE TITLE
v1.0.5 seems to return `undefined` for every file.

### ISSUE THREAD

**Issue by rwjblue** [stefanpenner/broccoli-persistent-filter#26]
```
ember new foo
ember build
cat dist/assets/foo.js
~~~~ snip ~~~~

define('foo/router', function () {

        'use strict';

        undefined

});
define('foo/app', function () {

        'use strict';

        undefined

});
```

**Comment by rwjblue** [comment]
I believe that this is because https://github.com/stefanpenner/broccoli-persistent-filter/blob/7f1de507ed6d952daa8a21e2e67720b7f834b534/index.js#L221-L231 does not return the `outputString` (so it returns `undefined`).

**Comment by stefanpenner** [comment]
@rwjblue i totally tested this locally and all appeared fine... I blame NPM

**Comment by rwjblue** [comment]
Published v1.0.6

**Comment by stefanpenner** [comment]
thanks

**Comment by stefanpenner** [comment]
more tests++

**Comment by rwjblue** [comment]
Released v1.0.7 with https://github.com/stefanpenner/broccoli-persistent-filter/commit/6a15d52b83fd2ee798eb3843201363dd02520e3d
